### DISCUSSION TITLE
Support of concatenation for SparseTensor

### DISCUSSION THREAD

**Discussion by jasperhyp** [pyg-team/pytorch_geometric#4420] [Ideas]
Can `SparseTensor` be adapted so that `torch.cat()` can be used, or can a concatenation function be added for `SparseTensor`?

**Comment by rusty1s** [comment]
There exists [`torch_sparse.cat`](https://github.com/rusty1s/pytorch_sparse/blob/master/torch_sparse/cat.py#L26), which can be used similarly to `torch.cat`. Hope this helps!

**Reply by jasperhyp** [reply]
Thanks so much!  I unfortunately did not find the documentation and the intro in GitHub repo does not contain this function...
